website/ref/language.md

```diff
@@ -2392,18 +2393,27 @@ itself or defined by the user.
 
 ### Importing modules with `use`
 
-Modules are imported using the `use` special command, which accepts a **module
-spec** and an optional alias:
+Modules are imported using the `use` special command. It requires a module spec
+and an optional alias:
 
 ```elvish
 use $spec $alias?
 ```
 
-Both the module spec and the alias must appear as a single string literal.
+The module spec and the alias must both be a [string literal](#string).
```

Comment:
Not sure why you removed "single" - it is needed to exclude cases where the spec of the alias appear as a compound of multiple string literals, which is not valid:

```
use 'a'/b/c
```

Reply:
I would argue that your example is a "string literal".; albeit not a "single" string literal. And it is not obvious to me why a "single" (which I interpret as "simple") string literal is relevant in this context. Also, neither your, or my, definition (with or without the "single" prefix) appears anywhere in the documentation AFAICT. Why can't a "single string literal" in this context be composed of a composition of string literals? This definition, obviously,

Reply:
I opted for something a bit more verbose with a link to the "Compounding" section.

Reply:
Thanks, the wording you chose is the indeed clearer.